packages/cli/openapi-ir-to-fern/src/converters/convertEndpoint.ts

```diff
@@ -64,7 +64,7 @@ export function convertEndpoint({
     }
 
     const convertedEndpoint: RawSchemas.HttpEndpointSchema = {
-        path: endpoint.path,
+        path: environments?.type === "single" ? `${environments.endpointPathPrefix}${endpoint.path}` : endpoint.path,
```

Comment:
do the urls need to be joined safely here? dont want a double slash somewhere

Reply:
Yeah I trim the trailing slash from the `endpointPathPrefix` https://github.com/fern-api/fern/blob/02169392c4dfd70a3ff50476425c8f6abb75f643/packages/cli/openapi-ir-to-fern/src/getEnvironments.ts#L88-L89